### ISSUE TITLE
Interpretation of results on the stump sequence

### ISSUE THREAD

**Issue by dragonlong** [nerfstudio-project/nerfstudio#1387]
**Describe the bug**
Just checked the nerfstudio paper https://arxiv.org/pdf/2302.04264.pdf, it seems the stump sequence fails (probably with some floaters) compared to others on the mip-nerf-360 datasets, while mip-nerf-360 paper can get a reasonable performance. I am wondering how to interpret this results, one thing I noticed that is on the sparser input training views. Besides that, is that the distortion loss not well performing, or we need some tricks like warm-up training, or MLP still performs better? Any hints are appreciated.

**To Reproduce**
Check Table below
<img width="756" alt="image" src="https://user-images.githubusercontent.com/8425887/217880680-bb00b9e9-1272-4d8b-afbf-0c4157868dea.png">

**Comment by dragonlong** [comment]
One add-up result with the nerfacto variant, it seems we can get better performance by simply turning off the distortion loss on **stump** sequence from mip-nerf-360 paper, boosting up from ~19dB to ~25dB, but other sequences slightly drop by ~1dB regarding PSNR. Personally I don't think PSNR is a good metric compared to LPIPS or other sharpness metrics, but from my visual check, at least the obvious floaters are gone, maybe @ethanweber can help check on this part.  Also is there any space that this distortion loss implementation can be improved? Thanks!

**Comment by tancik** [comment]
Yep, we found a few tricks that can get stump to work better, another one is to increase the `near plane`. However this isn't very general. We wanted to avoid having scene specific configs since the goal of nerfstudio is to be more general. On the scenes we have tested with, we have found that a little bit of distortion loss helps reduce fogginess.

> Also is there any space that this distortion loss implementation can be improved? Thanks!

Yea potentially! If people fine parameters / tricks that help nerfacto we will integrate them.
